.rubocop.yml

```diff
@@ -46,6 +46,9 @@ Lint/LiteralInInterpolation:
 Lint/NonLocalExitFromIterator:
   Enabled: false
 
+Style/AndOr:
+  EnforcedStyle: conditionals
+
```

Comment:
That's what I thought, too. But it's not default. If I run `bundle exec rake rubocop`  without those lines, I get this:

```
Offenses:

lib/rspec/core/formatters/failure_list_formatter.rb:51:43: C: Style/AndOr: Use || instead of or.
          bt        = exception.backtrace or return
                                          ^^
lib/rspec/core/formatters/failure_list_formatter.rb:53:54: C: Style/AndOr: Use || instead of or.
          bt_lines  = bt.reject { |l| exclude =~ l } or return
                                                     ^^
lib/rspec/core/formatters/failure_list_formatter.rb:57:64: C: Style/AndOr: Use || instead of or.
            md            = bt_line.match(/^(.+?):(\d+):(.*)/) or next
                                                               ^^
```

Reply:
Ah I see, no them, please remove this and use `||` we prefer `&&` and `||`

Reply:
Done